functests/4_latency/latency.go

```diff
@@ -114,66 +115,66 @@ var _ = Describe("[performance] Latency Test", func() {
+		// we can not use all isolated CPUs, because if reserved and isolated include all node CPUs, and reserved CPUs
+		// do not calculated into the Allocated, at least part of time of one of isolated CPUs will be used to run
+		// other node containers
+		// at least two isolated CPUs to run oslat + one isolated CPU used by other containers on the node = at least 3 isolated CPUs
+		if isolatedCpus.Size() < 3 {
+			Skip(fmt.Sprintf("Skip the oslat test, the profile %q has less than two isolated CPUs", profile.Name))
+		}
 
-			// verify the maximum latency only when it requested, because this value can be very different
-			// on different systems
-			if maximumLatency == -1 {
-				Skip("no maximum latency value provided, skip buckets latency check")
-			}
+		oslatArgs := []string{
+			fmt.Sprintf("-runtime=%s", latencyTestRuntime),
+		}
+		latencyTestPod = getLatencyTestPod(profile, workerRTNode, testName, oslatArgs)
+		createLatencyTestPod(latencyTestPod)
 
-			latencies := extractLatencyValues("oslat", `Maximum:\t*([\s\d]*)\(us\)`, workerRTNode)
+		// verify the maximum latency only when it requested, because this value can be very different
+		// on different systems
+		if maximumLatency == -1 {
+			Skip("no maximum latency value provided, skip buckets latency check")
+		}
 
-			// under the output of the oslat very often we have one anomaly high value, for example
-			// Maximum:    16543 15 15 14 13 12 12 13 12 12 12 12 12 12 12 12 12 (us)
-			// it still unclear if it oslat bug or the kernel one, but we definitely do not want to
-			// fail our test on it
-			var anomaly bool
-			for _, lat := range strings.Split(latencies, " ") {
-				if lat == "" {
-					continue
-				}
+		latencies := extractLatencyValues("oslat", `Maximum:\t*([\s\d]*)\(us\)`, workerRTNode)
 
-				curr, err := strconv.Atoi(lat)
-				Expect(err).ToNot(HaveOccurred())
+		// under the output of the oslat very often we have one anomaly high value, for example
+		// Maximum:    16543 15 15 14 13 12 12 13 12 12 12 12 12 12 12 12 12 (us)
+		// it still unclear if it oslat bug or the kernel one, but we definitely do not want to
+		// fail our test on it
+		var anomaly bool
+		for _, lat := range strings.Split(latencies, " ") {
+			if lat == "" {
+				continue
+			}
 
-				// skip the anomaly value
-				if curr > maximumLatency && !anomaly {
-					anomaly = true
-					continue
-				}
+			curr, err := strconv.Atoi(lat)
+			Expect(err).ToNot(HaveOccurred())
 
-				Expect(curr < maximumLatency).To(BeTrue(), "The current latency %d is bigger than the expected one %d", curr, maximumLatency)
+			// skip the anomaly value
+			if curr > maximumLatency && !anomaly {
+				anomaly = true
+				continue
 			}
-		})
-	})
+
+			Expect(curr < maximumLatency).To(BeTrue(), "The current latency %d is bigger than the expected one %d", curr, maximumLatency)
+		}
+	},
+		Entry("Valid test case #1", 0, true, 10, 10, true),
+		Entry("Valid test case #2", 0, false, 10, 10, true),
```

Comment:
Regarding the call here , even if I modified the inner function of describeTable , it will still not update the values of the variable